src/domain/common/whiteboard/whiteboard.guest-access.service.spec.ts

```
@@ -46,53 +48,53 @@ describe('WhiteboardGuestAccessService', () => {
-    } as unknown as jest.Mocked<AuthorizationPolicyService>;
+    } as unknown as Mocked<AuthorizationPolicyService>;
 
     communityResolverService = {
-      getCommunityFromWhiteboardOrFail: jest
+      getCommunityFromWhiteboardOrFail: vi
         .fn()
         .mockResolvedValue({ id: 'community-1' }),
-      getSpaceForCommunityOrFail: jest
+      getSpaceForCommunityOrFail: vi
         .fn()
         .mockResolvedValue(createSpace(true)),
```

ESLint

[error] Replace `⏎········.fn()⏎········` with `.fn()`

(prettier/prettier)